packages/phoenix-evals/src/phoenix/evals/models/anthropic.py

```diff
@@ -1,10 +1,16 @@
 from dataclasses import dataclass, field
-from typing import Any, Dict, List, Optional
+from typing import Any, Dict, List, Optional, Tuple
 
 from phoenix.evals.exceptions import PhoenixContextLimitExceeded
 from phoenix.evals.models.base import BaseModel
 from phoenix.evals.models.rate_limiters import RateLimiter
 
+MINIMUM_ANTHROPIC_VERSION = "0.18.0"
+
+
+def anthropic_version(version_str: str) -> Tuple[int, ...]:
```

Comment:
```suggestion
def anthropic_version(version_str: str) -> Tuple[int, int, int]:
```

Suggestion: Use the most specific return type possible.

Reply:
Because we can't guarantee the input string format, this fails a type check